src/Utils.hs

```diff
@@ -0,0 +1,13 @@
+module Utils (saltedRange) where
+
+
+-- Get random value in range based on salt value between 0.0 and 1.0.
+-- 0.0 mens minimal value.
+-- 1.0 means maximum value.
+-- Supposed to be used to get random value in specific range using item of
+-- infinite random values (number between 0.0 and 1.0) list.
+-- Usage examples:
+--   saltedRange 0.2 (20, 30) = 22.0
+--   saltedRange 1   (20, 30) = 30.0
```

Comment:
So, if I understand correctly, this function takes some factor in `[0.0, 1.0]` and projects it to some range `[a, b]`, right? So effectively it has nothing to do with salt or randomness by itself. It just projects a value to a range. Let's rename it to `projectToRange` or similar, then. And this is already a function I could call reusable across the code base. Please, correct me if any of my assumptions are wrong.

I remember that I reinvented something similar in https://github.com/tsoding/dimooper and it feels like a pretty common task. Such function should be somewhere in thirdparty utility libraries, but I'm failing to google anything.